test/integration/third-parties/test/index.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env jest */
```

Comment:
Consider using e2e tests so that it can check both dev and build mode easily, and we can make the routes name more meaningful like `/google-embed` and `/youtube-embed`

https://github.com/vercel/next.js/tree/canary/test/e2e/hello-world for pages
https://github.com/vercel/next.js/tree/canary/test/e2e/app-dir/hello-world for app router